tests/Stormpath/Tests/Cache/CacheTest.php

```diff
@@ -0,0 +1,70 @@
+<?php namespace Stormpath\Tests\Cache;
+
+
+use Stormpath\Tests\BaseTest;
+
+class CacheTest extends BaseTest
```

Comment:
These test looks good. However, I think we should add tests to check that things get deleted from the cache when the resource is deleted in Stormpath and also test to verify that we're not using cacheing at all when the `NullCacheManager` is specified. 

This should hopefully be the last thing before we merge to dev.